.travis.yml

```
@@ -46,7 +46,9 @@ script:
     - |
       if [ "${ARCH}" = "x86_64" ]; then
       sudo bash ./print_debug_info.sh &
-      travis_wait 60 sudo bash ./test_istgt.sh || travis_terminate 1;
+      ## If test takes more than 45 minutes then print tail logs in /tmp/istgt.log and terminate travis
+      ## Having 60minutes is causing travis timeout and not able to know easily which test is failed
+      travis_wait 45 sudo bash ./test_istgt.sh || sudo tail -500 /tmp/istgt.log || travis_terminate 1;
```

travis_terminate won't be executed in failure case. 
```suggestion
      travis_wait 45 sudo bash ./test_istgt.sh || ( sudo tail -500 /tmp/istgt.log && travis_terminate 1 );
```

---

Missied it

---

Done